web/modules/custom/portland_openid_connect/src/Routing/RouteSubscriber.php

```diff
@@ -60,7 +60,10 @@ protected function alterRoutes(RouteCollection $collection) {
     // Custom overrides on Pantheon dev, test, and live environments to enable OpenID login form
     if(
       isset($_ENV['PANTHEON_ENVIRONMENT']) &&
-      in_array($_ENV['PANTHEON_ENVIRONMENT'], ['dev', 'test', 'live'])
+      in_array($_ENV['PANTHEON_ENVIRONMENT'], [
+        // uncomment this line for local testing
+        //'lando', 
```

Comment:
@ccdatapdx , while reviewing this, I realized that we have code in porlandor to allow local testing of the button text change.